src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -467,14 +455,6 @@ MulticopterAttitudeControl::parameters_update()
 	param_get(_params_handles.yaw_rate_max, &_params.yaw_rate_max);
 	_params.mc_rate_max(2) = math::radians(_params.yaw_rate_max);
 
-	/* manual attitude control scale */
-	param_get(_params_handles.man_roll_max, &_params.man_roll_max);
```

Comment:
I found no corresponding PARAM_DEFINE statements for the deleted parameters. Nor did I find any actual usage of fields such as  _params.man_roll_max in this file.